webvtt/parsing/cue-text-parsing/dat/entities.dat

```diff
@@ -38,13 +38,13 @@
 a&lrm;b
 #errors
 #document-fragment
-| "a‎b"
+| "a\u200Eb"
 
 #data
 a&rlm;b
 #errors
 #document-fragment
-| "a‏b"
+| "a\u200Fb"
 
 #data
 &quot;
```

Comment:
This test (at least) is wrong per the current spec, since we changed to support all of HTML's charrefs.